crud/src/test/resources/ITCaseCrudResourceRDBMSTest/config/config.properties

```diff
@@ -1,3 +1,4 @@
 # set default isolation strategy for tests to SEMAPHORE else breakpoints in IDE
 # when debugging unit tests will timeout with default strategy (THREAD).
 hystrix.command.default.execution.isolation.strategy=SEMAPHORE
+execution.isolation.thread.timeoutInMilliseconds=30000
```

Comment:
The property `execution.isolation.thread.timeoutInMilliseconds` is not a valid configuration for hystrix based on the doc.  Per documentation the property name to configure the default behavior is `hystrix.command.default.execution.isolation.thread.timeoutInMilliseconds`.